### DISCUSSION TITLE
[FR] Day navigation buttons in main dashboard

### DISCUSSION THREAD

**Discussion by c0pper** [Freika/dawarich#631] [Ideas]
**Describe the bug**
This is not a bug, just a (maybe simple to implement) quality of life suggestion. Just add day navigation arrows like in the screen so it's easy to navigate the most recent days instead of manually selecting each day from the calendar...
![image](https://github.com/user-attachments/assets/4b537e30-4ac3-4ef2-a23f-acf2159e6d0d)

Thank you for dawarich!

**Comment by Andi-bz** [comment]
+1

**Comment by pablogglz** [comment]
+1

**Comment by Freika** [comment]
Done in 0.26.1
